singletons-base/CHANGES.md

```diff
@@ -13,24 +13,56 @@ Changelog for singletons-base project
     promoting and singling term-level definitions, but but nothing else. This
     library continues to require the latest stable release of GHC.
   * The `singletons-base` library defines promoted and singled versions of
-    definitions from the `base` library, which includes everything under
-    `Data.Singletons.Prelude.*`. This library continues to require the latest
-    stable release of GHC.
+    definitions from the `base` library, including the `Prelude`. This library
+    continues to require the latest stable release of GHC.
 
   Consult the changelogs for `singletons` and `singletons-th` for changes
   specific to those libraries. For more information on this split, see the
   [relevant GitHub discussion](https://github.com/goldfirere/singletons/issues/420).
 * Require building with GHC 8.12.
-* Due to the `singletons` package split, the `Data.Singletons.CustomStar` and
-  `Data.Singletons.TH` modules in `singletons-th` no longer re-export any
-  definitions from `Data.Singletons.Prelude.*`. The `singletons-base` library
-  now provides versions of these modules—`Data.Singletons.Prelude.CustomStar`
-  and `Data.Singletons.Prelude.TH`, respectively—that do re-export
-  definitions from `Data.Singletons.Prelude.*`.
+* The modules in `singletons-base` have been renamed to better reflect the
```

Comment:
I don't see a mention anywhere about the highly controversial `Data.Singletons.Base.Enum` module. Perhaps add it?

Reply:
I've added a bit below here that explains this a bit more.